packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -366,27 +400,36 @@ const PersonalDetailsForm = observer(() => {
                                             name='phone'
                                             id={'phone'}
                                             label={localize('Phone number*')}
-                                            className={clsx({
-                                                'account-form__fieldset--phone':
-                                                    account_settings?.phone_number_verification?.verified,
-                                            })}
                                             //@ts-expect-error type of residence should not be null: needs to be updated in GetSettings type
                                             value={values.phone}
+                                            hint={hintMessage({
+                                                is_phone_number_editted: account_settings.phone !== values.phone,
+                                                is_phone_number_empty: !account_settings.phone,
+                                            })}
                                             onChange={(e: ChangeEvent<HTMLInputElement>) => {
                                                 handleChange(e);
                                                 setStatus('');
                                             }}
                                             onBlur={handleBlur}
                                             required
                                             error={errors.phone}
-                                            disabled={isFieldDisabled('phone') || is_phone_field_disable}
+                                            disabled={
+                                                isFieldDisabled('phone') ||
+                                                is_request_button_disabled ||
+                                                !!next_request_time
```

Comment:
Can we clarify what next_request_time refers to? Is it the timer for the email attempt or the phone number attempt? If possible, can we rename this flag to specify which attempt it is associated with?

Reply:
sure let me separate these timer 🙏